(sync) Resolve conflicts with gatsby-i18n-source @ 79b09bc (#172)
gatsbyjs/gatsby-ja: docs/docs/gatsby-cli.md

  On Your Network:  http://192.168.0.212:8000/ // highlight-line
```

> <<<<<<< HEAD

Reviewer: ```suggestion
```

> **ヒント**: Windows では 0.0.0.0:8000 にアクセスできません。(ただし、localhost:8000 または「ネットワークの URL」を使うと問題なくアクセスできます）

Reviewer: ```suggestion
```

> =======

Reviewer: ```suggestion
```

> **Note**: To access Gatsby on your local machine, use either `http://localhost:8000` or the "On Your Network" URL.

Reviewer: ```suggestion
**ヒント**: ローカルにてビルドされた Gatsby サイトには、`localhost:8000` または `On Your Network` に表示された URL からアクセスできます。
```

> >>>>>>> 79b09bc29f133961f3d7de0f36a25ff727e6c22a

Reviewer: ```suggestion
```
